tests/support/helpers.py

```diff
@@ -685,7 +685,7 @@ def wrap(cls):
                             username
                         )
                     )
-                    create_user = cls.run_function('user.add', [username])
+                    create_user = cls.run_function('user.add', [username], **kwargs)
```

Comment:
Is the intention here just to pass `groups`? I'm wary of passing all kwargs unless that's truly needed.

Reply:
@cachedout Yes, this is to pass groups. I'll change it to be more specific.

Reply:
Actually, looking at this again. We are making the kwargs dictionary for the sole purpose of passing it to `user.add`. It is being used to pass timeout, groups, and password.